students/TacSPx/lesson03/list_lab.py

```diff
@@ -0,0 +1,187 @@
+# ---------------------------------------------------------------------------- #
+# Title: list_lab
+# Description: Four series of codes for modifying lists
+#
+# ---------------------------------------------------------------------------- #
+
+# series1:
+'''
+Step 1 Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”
+Step 2 Display the list (plain old print()
+Step 3 Ask the user for another fruit and add it to the end of the list and display list
+Step 4 Ask the user for a number and display the number back to the user with
+        the fruit corresponding to that number (on a 1-is-first basis)
+Step 5 Add another fruit to the beginning of the list using “+” and display the list
+Step 6 Add another fruit to the beginning of the list using insert() and display the list
+Step 7 Display all the fruits in the list that begin with the letter “P”, using a for loop
+'''
+
+def print_list():
+    print("The current list is: ", fruit)
+
+print()
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”
+fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+print_list() # Display the list using print_list function
+# Ask the user for another fruit and add it to the end of the list and display list
+fruit_response = input("Add another fruit to the 'end' of the list: ")
+print("You typed '{}'!".format(fruit_response))
+fruit.append(fruit_response.title())
+print_list()
+print()
+# Ask the user for a number and display the number back to the user with
+# the fruit corresponding to that number (on a 1-is-first basis)
+number_response = input("Pick a number in the list (1 - 5): ")
+print("You picked '{}'!".format(number_response))
+while True:
+    if (number_response == "1"):
+        print("Number 1 in the list = ", '"', fruit[0], '"')
+        break
+    elif (number_response == "2"):
+        print("Number 2 in the list = ", '"', fruit[1], '"')
+        break
+    elif (number_response == "3"):
+        print("Number 3 in the list = ", '"', fruit[2], '"')
+        break
+    elif (number_response == "4"):
+        print("Number 4 in the list = ", '"', fruit[3], '"')
+        break
+    elif (number_response == "5"):
+        print("Number 5 in the list = ", '"', fruit[4], '"')
+        break
+    else:
+        print("Error! you didn't pick a number between 1 and 5!")
+        break
```

Comment:
While this works, it breaks as the list grows.  Maybe another option would work?